### DISCUSSION TITLE
新的下載器開發計畫

### DISCUSSION THREAD

**Discussion by hgalytoby** [hgalytoby/MyselfAnimeDownloader#47] [General]
## 計畫背景 
現在我的程式設計能力與設計此專案的程式能力以不同。

我一直想開發一個新的下載器，但因為缺乏動力，一直遲遲未開始。

現在希望能找到志同道合的夥伴一起完成這個專案。

## 開發目標  
新的下載器應具備以下功能與特點：

**繼承現有功能**  
- 繼承下載器已有的功能。
- 能下載 Myself 網站的影片。

**功能擴展**  
- 新增支援其他動漫網站的能力(網址我就不附了)，例如：  
  - Anime1
  - Hanime1

**Web 化介面**  
- 將原來的下載器改成基於 Web 的方式運行。

### 額外建議  
**GUI 版本**  
- 若有需要 GUI，考慮使用 [Pywebview](https://github.com/r0x0r/pywebview) 偽裝成桌面應用程式。

**技術棧**  
- 前端：使用 [Vue.js](https://vuejs.org)  
- 後端：使用 [FastAPI](https://fastapi.tiangolo.com)

---

希望能聽聽大家的想法或其他建議！如果對這個專案感興趣，請隨時回覆參與討論 🙌

**Comment by Samuelcodeforfood** [comment]
我有興趣，大大的動力還在嗎

**Reply by hgalytoby** [reply]
有，如果要開發我是想用閒暇時間開發

**Reply by Samuelcodeforfood** [reply]
沒問題啊~ 我現在也有點忙學校作業，但是很有興趣
請問要怎麼聯絡你

**Reply by hgalytoby** [reply]
3個人在開坑
